pkg/lambdacomponents/defaults.go

```diff
@@ -29,7 +29,7 @@
  
  // LambdaComponents returns a set of stripped components used by the
```

Comment:
Can you change the comment here to be `Components returns ....` Some editor/linter (e.g. vscode) will show a warning when first word in comment is not same as func name.

Reply:
Done